source/users.rst

```diff
@@ -3,10 +3,16 @@
 Users
 ==============================
 
-The Operator requires
-Kubernetes Secrets to be deployed before the PXC Cluster is started. The name of the
-required secrets can be set in ``deploy/cr.yaml`` under the
-``spec.secretsName`` section.
+MySQL user accounts within the Cluster can be divided into two different groups:
+
+* application-level (unprivileged) users,
```

Comment:
Instead of using parentheticals, lets instead change this formatting to be more readable.  Something like:

* Type - Description
* Type: Description

or similar